client/tests/integration/smartcontracts/Cargo.toml

```
@@ -25,3 +25,4 @@ codegen-units = 1   # Further reduces binary size but increases compilation time
 
 [workspace.dependencies]
 iroha_wasm = { path = "../../../../wasm", features = ["debug"]}
+iroha_validator = { path = "../../../../wasm/validator", features = ["debug"]}
```

```suggestion
iroha_validator = { path = "../../../../wasm/validator"}
```

it's already set by `iroha_wasm`